secretmanager/runtime/src/main/java/io/quarkiverse/googlecloudservices/secretmanager/runtime/SecretManagerException.java

```diff
@@ -0,0 +1,22 @@
+package io.quarkiverse.googlecloudservices.secretmanager.runtime;
+
+public class SecretManagerException extends RuntimeException {
```

Comment:
This is only used by the `SecretManagerCredentialProvider` so better named it `SecretManagerCredentialProviderException`.

Reply:
Renamed in 68bb8fe

Reply:
Sorry, it misses a **s** => SecretManagerCredential**s**ProviderException